### ISSUE TITLE
Command name can be elided in consecutive commands

### ISSUE THREAD

**Issue by Michael-F-Bryan** [Michael-F-Bryan/gcode-rs#3]
If you are repeating a command, just with different arguments, the command's name can be elided.

e.g.

```text
G01 X1.0 Y2.0
G01 Z10.0          <- "G01" can be elided here
```

This is currently breaking the integration tests because it's used all over the place in hand-written code :(

**Comment by Michael-F-Bryan** [comment]
For now (f9a2d1e) I've commented out lines which elide the command name. Later on when I deal with this I'll fix it up again.

**Comment by Michael-F-Bryan** [comment]
Here's a sample file:


[Insulpro Piping - 115mm OD - 40mm WT.txt](https://github.com/Michael-F-Bryan/gcode-rs/files/3434009/Insulpro.Piping.-.115mm.OD.-.40mm.WT.txt)

**Comment by Michael-F-Bryan** [comment]
Fixed in #38.
